evalml/tests/data_checks_tests/test_class_imbalance_data_check.py

```diff
@@ -46,10 +46,10 @@ def test_class_imbalance_data_check_binary(input_type):
         y_balanced = y_balanced.to_numpy()
 
     elif input_type == "ww":
-        X = ww.DataTable(X)
-        y = ww.DataColumn(y)
-        y_long = ww.DataColumn(y_long)
-        y_balanced = ww.DataColumn(y_balanced)
+        X.ww.init()
```

Comment:
O interesting, do we need to set `X = X.ww.init()` here?

Reply:
(Just wondering between the difference between this and line 114)

Reply:
+1, my guess is that the assignment in `X = X.ww.init()` is redundant but harmless

Reply:
Great question! We can't do assign the result of `ww.init` to anything because `ww.init` returns `None`

Reply:
interesting that `ww.init()` modifies in place while the equivalent for series returns a new series

Reply:
I get what you mean! There is `series.ww.init()` the problem is that if the logical type doesn't match the physical type of the series, it will error out. `ww.init_series` will actually do the type conversion if needed.

![image](https://user-images.githubusercontent.com/41651716/115887863-58c2c400-a420-11eb-82aa-85c4e07b7ca2.png)

Reply:
Got it, thanks for the explanation!

Reply:
> Great question! We can't assign the result of `ww.init` to anything because `ww.init` returns `None`. Just fixed a typo about this in this file that I missed.

@freddyaboulton  So, I'm a little late to the party here, but I'm wondering if we should actually return `self` from our Woodwork init method. If I'm thinking about this correctly, doing so would allow for chaining operations like `df.ww.init().ww.describe()` and also make the re-assignment situation valid. Thoughts?